### DISCUSSION TITLE
How to integrate other language for text search config

### DISCUSSION THREAD

**Discussion by taishi55** [supabase/supabase#10164] [Questions] [answered]
currently the default language for textsearch query is `english`

I want to know how to setup other languages in the text search config.

**Comment by steve-chavez** [comment, marked as answer]
You can change it as mentioned herehttps://supabase.com/docs/reference/javascript/textsearch#text-search

```js
const { data, error } = await supabase
  .from('quotes')
  .select('catchphrase')
  .textSearch('catchphrase', `'fat' & 'cat'`, {
    config: 'another-lang'
  })
```

**Reply by taishi55** [reply]
I tried that, but I got an error saying the config is not predefined. I think `english` is pre-installed as default but other language is not installed. so the changing the config parameter is not enough...

**Reply by steve-chavez** [reply]
Hm, I see. Then that's a different issue that should be reported on https://github.com/supabase/postgres (allow other dictionaries)
